ravenframework/SupervisedLearning/ARMA.py

```diff
@@ -252,6 +252,11 @@ class cls.
     domainEnumType = InputTypes.makeEnumType('domain', 'truncateDomainType', ['positive', 'negative'])
     outTrunc.addParam('domain', domainEnumType, True)
     specs.addSub(outTrunc)
+
+    clusterEvalModeEnum = InputTypes.makeEnumType('clusterEvalModeEnum', 'clusterEvalModeType', ['clustered', 'truncated', 'full'])
+    specs.addSub(InputData.parameterInputFactory('clusterEvalMode', contentType=clusterEvalModeEnum)) # for pickled ROMCollection
+    specs.addSub(InputData.parameterInputFactory('maxCycles', contentType=InputTypes.IntegerType)) # for pickled Interpolated ROMCollection
+
```

Comment:
Could you update it with descriptions? The following are from pickledRom.py:
![image](https://user-images.githubusercontent.com/7321071/212979883-cdf900e9-8d95-4881-bdaf-b17d93b1b690.png)

Reply:
Right now the connection to actually use clusterEvalMode in ARMA is broken, so I have other things to do before that part can be usefully merged.

Reply:
I split the code for this into a branch: re_add_to_spec_full, which will be a different pull request.